CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.40.0
```

Comment:
we should have a changelog for https://github.com/aws/amazon-ecs-agent/pull/2452